### ISSUE TITLE
CacheDB 报错 headers 未定义

### ISSUE THREAD

**Issue by FloatSheep** [CrazyCreativeDream/CacheDB#2]
控制台输出

`cache-db.min.js:1 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'headers')
    at cache-db.min.js:1:459`

调用方法

```javascript
async function checkAndInitializeVersionKey() {
  try {
    let versionExists;
    try {
      versionExists = await cacheObject.read("version");
    } catch (readError) {
      console.log("读取version键时发生错误:", readError);
      versionExists = null;
    }

    if (versionExists === null || versionExists === undefined) {
      await cacheObject.write("version", "0.0.0-0");
      console.log("Version 初始化成功。");
    }
    return Promise.resolve();
  } catch (error) {
    console.error("初始化或读取version键时发生错误:", error);
    return Promise.resolve();
  }
}
```

这段代码的本意是尝试读取一个键，如果报错就新建，但是由于这个报错无法被 catch 捕获（try catch 和 promise 的链式调用均无法捕获），逻辑就跑不起来了

![image](https://github.com/user-attachments/assets/f9e7c7b2-c3cd-4f4b-802a-08aa1af39807)

**Comment by ChenYFan** [comment]
收到，晚上加方法

**Comment by ChenYFan** [comment]
![image](https://github.com/user-attachments/assets/7c782a91-da60-4a55-806d-a7fcd2d81db8)
![image](https://github.com/user-attachments/assets/bf93fa0e-b3b4-4cd6-a745-8291a7b49eb7)

**Comment by ChenYFan** [comment]
added in #[e0e3025](https://github.com/CrazyCreativeDream/CacheDB/commit/e0e3025fbeed1c098401443c3e2b3a3145119294) will published in 1.1.5
